src/store/todo.store.js

```diff
@@ -0,0 +1,68 @@
+import { setTodos } from '../service/todo.service.js';
+import { changeFilterList } from '../ui/todo/todo.dom.js';
+export const addTodo = function (value) {
+  const timestamp = Date.now();
+  const todo = {
+    key: timestamp,
```

Comment:
[Nit] timestamp도 대부분의 상황에서는 유니크한 값이지만 좀 더 안전하게 유니크 값을 부여하면 좋을 것 같습니다.

```suggestion
    key: generateId('todoitem'),
```